### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by anhthuqs99**
Add the unit test

### CODE REVIEW

File: webgiasu/tutor/tests.py [hunk 1]
```diff
@@ -1,3 +1,48 @@
+from calendar import calendar
```
Comment (tsthien) [inline webgiasu/tutor/tests.py]: ??what is this

File: webgiasu/tutor/tests.py [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+        self.assertEqual(post.calendar, res['calendar'])
+        self.assertEqual(post.salary, res['salary'])
+        self.assertEqual(post.contact, res['contact'])
+        self.assertEqual(post.date_posted, res['date_posted'])
+        self.assertEqual(post.date_outdate, res['date_outdate'])
+    
+    def test_findtutorpost(self):
+        res = self.arg.copy()
+        self.compare_with(res)
 # Create your tests here.
```
Comment (tsthien) [inline webgiasu/tutor/tests.py]:
```suggestion
Delete
```

File: webgiasu/employee/tests.py [hunk 3]
```diff
@@ -1,3 +1,44 @@
+from calendar import calendar
```
Comment (tsthien) [inline webgiasu/employee/tests.py]: Why did you need this lib

File: webgiasu/tutor/tests.py (hunk 1 above)
Comment (anhthuqs99) [reply webgiasu/tutor/tests.py]: deleted!

File: webgiasu/tutor/tests.py (hunk 2 above)
Comment (anhthuqs99) [reply webgiasu/tutor/tests.py]: deleted!

File: webgiasu/employee/tests.py (hunk 3 above)
Comment (anhthuqs99) [reply webgiasu/employee/tests.py]: wrong include. has deleted
